GCRCatalogs/agn.py

```diff
@@ -38,28 +38,51 @@ def __del__(self):
         quantity_modifiers = {
-            'blackHoleEddingtonRatio': 'blackHoleEddingtonRatio',
-            'blackHoleMass':           'blackHoleMass',
-            'dec':                     'dec',
-            'galaxy_id':               'galaxy_id',
-            'halo_mass':               'halo_mass',
-            'is_central':              'is_central',
-            'ra':                      'ra',
-            'redshift':                'redshift',
-            'redshift_true':           'redshift',
+            'blackHoleEddingtonRatio':                      'blackHoleEddingtonRatio',
+            'blackHoleAccretionRate':                       'blackHoleAccretionRate',
+            'blackHoleMass':                                'blackHoleMass',
+            'dec':                                          'dec',
+            'galaxy_id':                                    'galaxy_id',
+            'halo_mass':                                    'halo_mass',
+            'is_central':                                   'is_central',
+            'ra':                                           'ra',
+            'redshift':                                     'redshift',
+            'Av':                                           'Av',
+            'stellar_mass':                                 'stellar_mass',
+            'Mag_true_i_agnonly_no_agn_extinction_lsst_z0': 'M_i', 
+            'Mag_true_i_agnonly_lsst_z0':                   'M_i(extincted)',
         }
 
         # magnitudes
         for band in ['u', 'g', 'r', 'i', 'z', 'y']:
             quantity_modifiers['mag_{}_noagn_lsst'.format(band)] = 'mag_{}_lsst(galaxy)'.format(band)
-            quantity_modifiers['agn_{}_lsst'.format(band)] = 'mag_{}_lsst(agn)'.format(band)
-            quantity_modifiers['mag_{}_lsst'.format(band)] = (_calc_mag_sum,
+            quantity_modifiers['mag_{}_agnonly_no_agn_extinction_lsst'.format(band)] = 'mag_{}_lsst(agn)'.format(band)
+            quantity_modifiers['mag_{}_agnonly_lsst'.format(band)] = 'mag_{}_lsst(agn_extincted)'.format(band)
+            quantity_modifiers['mag_{}_no_agn_extinction_lsst'.format(band)] = (_calc_mag_sum,
                                                               'mag_{}_lsst(galaxy)'.format(band),
                                                               'mag_{}_lsst(agn)'.format(band),
                                                               )
+            quantity_modifiers['mag_{}_lsst'.format(band)] = (_calc_mag_sum,
+                                                              'mag_{}_lsst(galaxy)'.format(band),
+                                                              'mag_{}_lsst(agn_extincted)'.format(band),
+                                                              )
+        # Magnitudes
+        for band in ['g', 'r', 'i']:
+            quantity_modifiers['Mag_true_{}_noagn_lsst_z0'.format(band)] = 'Mag_true_{}_lsst_z0'.format(band)
 
+        for band in ['i']:
```

Comment:
Why this derived quantities has only i band? Can you justify (and potentially put a comment in the code for future reference)? Also, if in the future there will be no more band for these quantities, we should not use the for statement here.

Reply:
The catalog currently supplies only i-band rest-frame AGN magnitude. That may change.
Comment added.